app/assets/stylesheets/overrides/breadcrumb.scss

```diff
@@ -0,0 +1,5 @@
+.breadcrumb {
+  background-color: $white;
```

Comment:
I think we still want it offset some, just not quite as dark

Reply:
Oh okay, I restored the padding and changed the background color to be an off-white. The screenshot in the description is updated to reflect that. Is that better?